packages/http/httpcall_client.js

```diff
@@ -173,6 +173,19 @@ HTTP.call = function(method, url, options, callback) {
       }
     };
 
+    // Allow custom control over XHR and abort early.
+    if(options.beforeSend) {
+      // Sanity
+      var beforeSend = _.once(options.beforeSend);
+
+      // Possibly change the call so the xhr is the context.
```

Comment:
I don't understand what this comment means.  You're providing `null` as the context.

I think passing arguments to callbacks as arguments rather than as `this` is generally clearer (and we're moving away from the other style in Meteor).